apps/asap-server/src/utils/migrations.ts

```diff
@@ -1,30 +1,37 @@
 import { Results, Squidex } from '@asap-hub/squidex';
 import { Entity, Rest } from '@asap-hub/squidex/src/entities/common';
+import logger from './logger';
 
 export const applyToAllItemsInCollection = async <
   T extends Entity & Rest<unknown>,
 >(
   entityName: string,
   processEntity: (entity: T, squidexClient: Squidex<T>) => Promise<void>,
+  filter?: string,
 ): Promise<void> => {
   const squidexClient = new Squidex<T>(entityName, {
     unpublished: true,
   });
 
   let pointer = 0;
+  let processedItems = 0;
   let result: Results<T>;
 
   do {
     result = await squidexClient.fetch({
       $top: 10,
       $skip: pointer,
       $orderby: 'created asc',
+      $filter: filter,
     });
 
     for (const item of result.items) {
       await processEntity(item, squidexClient);
+      processedItems += 1;
     }
 
     pointer += 10;
-  } while (pointer < result.total);
+  } while (pointer <= result.items.length);
```

Comment:
Will changing this mean old migrations fail to work because they will be relying on the total property?